gengokit/httptransport/templates_test.go

```diff
@@ -173,12 +175,9 @@ func DecodeHTTPSumZeroRequest(_ context.Context, r *http.Request) (interface{},
 		fmt.Printf("Error while reading path params: %v\n", err)
 		return nil, errors.Wrap(err, "couldn't unmarshal path parameters")
 	}
-	queryParams, err := QueryParams(r.URL.Query())
+
+	queryParams := r.URL.Query()
```

Comment:
Is assigning this to a variable really even necessary? It was with the old `QueryParams` function, but seems not so now.